table_row.py

```diff
@@ -157,6 +157,7 @@ def merge_right(self):
         self.left_button.setEnabled(False)
                 
         # Table isn't gonna repaint itself. Gotta show users the changes we just made.
+        self.table.clearSelection()
```

Comment:
row merge buttons left selection shown, this clears it so you see the light blue color indicating the merge was done.